index.js

```diff
@@ -0,0 +1,23 @@
+const fs = require('fs');
+
+const fieldsToIgnore = ["NODE_CONFIG_DIR"];
```

Comment:
Probably want to allow this to be configured externally - for example if I'm testing locally, I want the config var to be set by me, and not passed in...

Reply:
Good point. Will apply that.

Reply:
Done! Thanks.